docs/content/preview/admin/yb-admin.md

```diff
@@ -2057,29 +2057,6 @@ yb-admin \
 * *replication_name*: The name of the replication to be enabled or disabled.
 * `0` | `1`: Disabled (`0`) or enabled (`1`). Default is `1`.
 
-#### change_xcluster_role
-
-Sets the xCluster role to `STANDBY` or `ACTIVE`.
-
-**Syntax**
-
-```sh
-yb-admin \
-    -master_addresses <master-addresses> \
-    change_xcluster_role \
-    <role>
-```
-
-* *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`. These are the addresses of the master nodes where the role is to be applied. For example, to change the target to `STANDBY`, use target universe master addresses, and to change the source universe role, use source universe master addresses.
-* *role*: Can be `STANDBY` or `ACTIVE`.
-
-**Example**
-
-```sh
-./bin/yb-admin \
-    -master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 \
-    change_xcluster_role STANDBY
-```
 
 #### get_xcluster_safe_time
```

Comment:
why are these xcluster commands mixed in with cdc commands

Reply:
These are old xCluster commands. The cdc commands have cdcsdk in them.